db/schema.rb

```diff
@@ -162,9 +162,9 @@
     t.string "why_dont_you_know_la"
     t.integer "unitletas"
     t.integer "builtype"
+    t.datetime "property_void_date"
     t.bigint "owning_organisation_id"
     t.bigint "managing_organisation_id"
-    t.datetime "property_void_date"
```

Comment:
Grrr, is this a problem, or should I remove from this PR?

Reply:
Preferable would be to remove from this PR I think and check whether it's there on a fresh `rake db:drop; rake db:create; rake db:migrate`. i.e. is the repo schema out of date or is this a local inconsistency.

Reply:
Still there when do the above, but have removed change from this PR.